plugins/dell-dock/fu-dell-dock-common.h

```diff
@@ -22,17 +22,23 @@
 #include "fu-dell-dock-i2c-ec.h"
 #include "fu-dell-dock-i2c-mst.h"
 #include "fu-dell-dock-i2c-tbt.h"
+#include "fu-dell-dock-usb-usb4.h"
 #include "fu-dell-dock-hub.h"
 #include "fu-dell-dock-hid.h"
 #include "fu-dell-dock-status.h"
 
-#define 	DELL_DOCK_EC_INSTANCE_ID	"USB\\VID_413C&PID_B06E&hub&embedded"
-#define 	DELL_DOCK_TBT_INSTANCE_ID	"TBT-00d4b070"
+#define  DELL_DOCK_EC_INSTANCE_ID     "USB\\VID_413C&PID_B06E&hub&embedded"
+#define  DELL_DOCK_TBT_INSTANCE_ID    "TBT-00d4b070"
+#define  DELL_DOCK_USB4_INSTANCE_ID   "TBT-00d4b071"
 
-gboolean	fu_dell_dock_set_power		(FuDevice *device,
-						 guint8 target,
-						 gboolean enabled,
-						 GError **error);
-void		 fu_dell_dock_will_replug	(FuDevice *device);
+#define  GR_USB_VID  0x8087
+#define  GR_USB_PID  0x0B40
 
-void		 fu_dell_dock_clone_updatable	(FuDevice *device);
+gboolean fu_dell_dock_set_power       (FuDevice *device,
+                                       guint8 target,
+                                       gboolean enabled,
+                                       GError **error);
+
+void     fu_dell_dock_will_replug     (FuDevice *device);
+
+void     fu_dell_dock_clone_updatable (FuDevice *device);
```

Comment:
the changes to this file should just be additions in this commit, if the whitespace needs adjusting do it in a different commit